pkg/bpfassets/attacher/bcc_attacher.go

```diff
@@ -129,6 +129,8 @@ func AttachBPFAssets() (*BpfModuleTables, error) {
 	bpfModules.Table = table
 	bpfModules.CPUFreqTable = cpuFreqTable
 
+	HardwareCountersEnabled = true
```

Comment:
turn it off if perf counters are not enabled [here](https://github.com/sustainable-computing-io/kepler/blob/f7dc88578aa311b8751350a035d6ab4fd3b84248/pkg/bpfassets/attacher/bcc_attacher.go#L99)

Reply:
@rootfs we have multiple counters
https://github.com/sustainable-computing-io/kepler/blob/f7dc88578aa311b8751350a035d6ab4fd3b84248/pkg/bpfassets/attacher/bcc_attacher.go#L89

so which one should I use for `HardwareCountersEnabled ` ? 
or should I set to false if any of the counter in the list is not enabled?

Reply:
set it to false as long as one of the counters is not enabled